.script/ReadMe.md

```diff
@@ -42,6 +42,7 @@ By default, the logs color is white. In case you want another color you can use
      - onFinalFailed:  error behavior at the end of the validation, execute in case one of the validation failed
    - Call runCheckOverChangedFiles with properties:
      - checkOptions:  the object you create in the earlier step 2  
+     - fileKinds: list of what file kinds to filter by. There are 3 kinds- "Added", "Modified", "Deleted"

```

Comment:
nit: consider to have the same syntax for all propeties. 
update the other to be like the new or vice versa.